### DISCUSSION TITLE
Add members to group using emails AND/OR send an invitation if not a member

### DISCUSSION THREAD

**Discussion by favourch** [outline/outline#3452] [Ideas]
Add members to group using emails AND/OR send an invitation if not a member.

I use outline to manage a small learning program for 1000 university students and currently finding it challenging to add multiple members to a group. I would appreciate if administrators are able to add multiple members using emails all at once than individually selecting members to add.

**Comment by Limezy** [comment]
I would advise to use a tool such as Dex to bridge your LDAP (or other technology, you can look at Dex's long list of connectors) user base to Outline's OIDC authentication method
